src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/IntegrationTestBase.java

```diff
@@ -198,14 +244,48 @@ public EntityModel createEntity() {
         return null;
     }
 
-    public void deleteEntity() {
-
+    public void deleteEntity(EntityModel entityModel) {
+        entityGenerator.deleteEntityModel(entityModel);
     }
 
+    /**
+     * This method will create a new user with hardcoded user info -John Doe
+     */
     public void createUSer() {
-        Entity newUser = Entity.USER_ITEM;
+        String postUrl = connectionSettings.getConnectionSettings().getBaseUrl() + "/api/shared_spaces/" +
```

Comment:
This is nice, this is how you would do it w/o having any sdk.
But we do have an sdk for Octane, which would propbably make this a bit simpler.
Try creating the user as an EntityModel object (from sdk) and using octane.entityList(users).create